bin/release

```diff
@@ -71,7 +71,7 @@ set -xe
 wget https://helm.astronomer.io/index.yaml -O /tmp/index.yaml.current
 helm repo index . --url https://helm.astronomer.io --merge /tmp/index.yaml.current
 gsutil cp -a public-read $HELM_CHART_PATH gs://helm.astronomer.io
-gsutil cp -a public-read -h "Cache-Control:no-cache,max-age=0" ./index.yaml gs://helm.astronomer.io
```

Comment:
This one seems like a regression -- when putting the index.yaml we _want_ to set max-age=0

Reply:
@sjmiller609 any idea here?

Reply:
@sjmiller609 says the header does not have any affect.